2019-01-21 17:58:04,808 ERROR: Exception on /view_people/2 [GET] [in /home/gene/.local/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/gene/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/gene/.local/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/routes.py", line 101, in view_people
    return render_template('view_people.html', title = 'View Person', select_people=select_people, form=form, notes_form=notes_form, viewpnotes=pnotes)
  File "/home/gene/.local/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/templates/view_people.html", line 80, in template
    {% endblock %}
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "/home/gene/.local/lib/python3.6/site-packages/jinja2/parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'.
2019-01-21 17:58:41,685 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 17:59:44,285 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:17:39,354 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:18:46,697 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:21:50,196 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:27:44,631 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:44:23,047 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:45:15,808 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:45:45,587 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:49:09,893 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:50:12,229 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:50:34,815 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:51:11,630 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:51:26,841 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:52:13,643 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:52:34,912 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:52:53,645 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:53:10,960 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 18:59:13,278 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:03:19,073 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:12:02,111 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:13:08,882 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:24:05,669 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:24:51,655 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:26:13,078 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:28:32,734 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:39:37,118 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:44:35,690 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-21 19:45:42,315 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 17:54:21,613 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 17:54:46,800 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 17:55:11,942 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 17:57:09,533 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 17:58:34,056 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 17:59:37,497 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:02:17,918 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:03:13,887 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:04:05,269 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:04:56,284 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:07:14,902 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:09:00,900 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-22 18:10:00,258 INFO: ReDefined CRM Startup [in /home/gene/Desktop/DESKTOP/DESKTOP/CRM/app/__init__.py:53]
2019-01-23 18:54:56,859 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 18:56:06,327 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 18:57:46,853 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 18:58:24,722 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 18:59:06,759 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 18:59:39,556 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:00:01,761 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:00:10,520 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:00:36,833 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:01:23,050 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:03:46,166 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:04:21,147 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:04:44,537 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:05:20,589 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-23 19:08:11,773 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
